docs/src/apalache/tuning.md

```diff
@@ -20,53 +20,44 @@ The following options are supported:
 1. __Randomization__: `smt.randomSeed=<int>` passes the random seed to `z3` (via
    `z3`'s parameters `sat.random_seed` and `smt.random_seed`).
 
-1. __Timeouts__: `search.smt.timeout=<seconds>` defines the timeout to the SMT
-   solver in seconds. The default value is `0`, which stands for the unbounded
-   timeout.  For instance, the timeout is used in the following cases: checking
-   if a transition is enabled, checking an invariant, checking for deadlocks.
-   If the solver times out, it reports 'UNKNOWN', and the model checker reports
-   a runtime error.
-
-1. __Invariant mode__: `search.invariant.mode=(before|after)` defines the moment
-   when the invariant is checked. In the `after` mode, all transitions are first
-   translated, one of them is picked non-deterministically and then the
-   invariant is checked. Although this mode reduces the number of SMT queries,
-   it usually requires more memory than the `before` mode. In the `before` mode,
-   the invariant is checked for every enabled transition independently. The
-   `before` mode may drastically reduce memory consumption, but it may take
-   longer than the `after` mode, provided that Apalache has enough memory. The
-   default mode is `before`.
-
-1. __Guided search__: `search.transitionFilter=<regex>`. Restrict the choice of
-   symbolic transitions at every step with a regular expression. The regular
-   expression should recognize words over of the form 's->t', where `s` is a
-   regular expression over step numbers and `t` is a regular expression over
-   transition numbers. For instance,
+2. __Timeouts__: `search.smt.timeout=<seconds>` defines the timeout to the SMT solver in seconds. The default value
+   is `0`, which stands for the unbounded timeout. For instance, the timeout is used in the following cases: checking if
+   a transition is enabled, checking an invariant, checking for deadlocks. If the solver times out, it reports '
+   UNKNOWN', and the model checker reports a runtime error.
+
+3. __Invariant mode__: `search.invariant.mode=(inLoop|atOnce)` defines the moment when the invariant is checked. In
+   the `atOnce` mode, all transitions are first translated, one of them is picked non-deterministically and then the
+   invariant is checked. Hence, they are checked "all at once". Although this mode reduces the number of SMT queries, it
+   usually requires more memory than the `inLoop` mode. In the `inLoop` mode, the invariant is checked for every enabled
```

Comment:
I think this needs some reference to the transition explorer to be understandable.
E.g., "is picked non-deterministically" is only meaningful in context.